src/components/themed/ShareButtons.tsx

```diff
@@ -34,10 +35,10 @@ function ShareButton(props: { text: string; onPress: () => Promise<void>; icon:
   const [pending, handlePress] = usePendingPress(onPress)
 
   return (
-    <TouchableOpacity accessible={false} style={styles.button} onPress={handlePress} disabled={pending}>
+    <EdgeTouchableOpacity accessible={false} style={styles.button} onPress={handlePress} disabled={pending}>
```

Comment:
This guy has also incorporated the `usePendingPress` hook.

Reply:
This is the only one I can 100% update without changing visual behavior.